### PR TITLE
[MetricsAdvisor] last steps deprecation

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

**Review by swathipil — APPROVED** [review]
lgtm!

File: sdk/metricsadvisor/ci.yml [hunk 1]
```diff
@@ -29,7 +29,3 @@ extends:
   parameters:
     ServiceDirectory: metricsadvisor
     TestProxy: true
-    Artifacts:
-    - name: azure-ai-metricsadvisor
-      safeName: azureaimetricsadvisor
-
```
Comment (kristapratico) [inline sdk/metricsadvisor/ci.yml:35]: Will this silently work? I have no idea

**Review by kristapratico — APPROVED** [review]

File: sdk/metricsadvisor/ci.yml (hunk 1 above)
Comment (swathipil) [reply sdk/metricsadvisor/ci.yml:35]: Looks like it does: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4083031&view=logs&j=8e5196d1-9c2a-56bb-41fd-e329e85c48fa&t=e2ecfcdb-d2d5-5b7d-c312-a514a9344a8c&l=13

**Comment by bachhoant** [comment]
087ca39e65ccbc9f7941c6881a86f8d3e66a51a6

**Pull request merged**